challenge-13/challenge-13.js

```diff
@@ -1,150 +1,180 @@
+    */
+    console.log( '\nMais estados adicionados:' );
+    brasil.unshift( 'Amazonas', 'Acre', 'Amapá' );
+    console.log( brasil );
+
+    /*
+    Remova o primeiro estado do array `brasil` e mostre-o no console.
+    */
+    console.log( '\nEstado removido:' );
+    console.log( brasil.shift( 0 ) );
+
+    /*
+    Crie um novo array chamado `newSul`, que receba somente os estados do sul,
+    pegando do array `brasil`. Não remova esses itens de `brasil`.
+    */
+    var newSul = brasil.slice( 2, 5 );
+
+    /*
+    Mostre no console os estados que estão em `newSul`.
+    */
+    console.log( '\nEstados do Sul do Brasil:' );
+    console.log( newSul );
+
+    /*
+    Mostre no console todos os estados que estão em `brasil`.
+    */
+    console.log( '\nAlguns Estados do Brasil:' );
+    console.log( brasil );
+
+    /*
+    Crie um novo array chamado `nordeste`, que tenha os estados do nordeste.
+    */
+    var nordeste = [ 'Alagoas', 'Bahia', 'Ceará', 'Maranhão', 'Paraíba', 'Pernambuco', 'Piauí', 'Rio Grande do Norte', 'Sergipe' ];
+
+    /*
+    Mostre no console os estados do nordeste.
+    */
+    console.log( '\nEstados do Nordeste:' );
+    console.log( nordeste );
+
+    /*
+    Remova de `brasil` os estados do `sudeste`, colocando-os em uma variável
+    chamada `newSudeste`.
+    */
+    var newSudeste = brasil.splice( -4, 4 );
+
+    /*
+    Adicione os estados do `nordeste` ao array `brasil`. Esses estados devem
+    ficar no mesmo nível que os estados já existentes, não em um array separado.
+    */
+    brasil.concat( nordeste );
+
+    /*
+    Mostre no console os estados em `newSudeste`.
+    */
+    console.log( '\nEstados em newSudeste:' );
+    console.log( newSudeste );
+
+    /*
+    Mostre no console os estados do `brasil`.
+    */
+    console.log( '\nAlguns estados do Brasil:' );
+    console.log( brasil );
+
+    /*
+    usando forEach, percorra o array `brasil` e gere um novo array chamado
+    `newBrasil`. Esse array deve ter cada item como um objeto, com as
+    propriedades:
+    - `id`: que será o índice do array `brasil`,
+    - `estado`: que será o estado do array `brasil`.
+    */
+    var newBrasil = [];
+    brasil.forEach( function( item, index ){
+        newBrasil.push( { id: index, estado: item } );
+    });
+
+    /*
+    Mostre o array `newBrasil` no console
+    */
+    console.log( '\nnewBrasil:' );
+    console.log( newBrasil );
+
+    /*
+    Percorra o array `brasil` e verifique se os estados tem mais de 7 letras cada,
+    atribuindo o resultado à uma variável. Se tiver, mostre no console a frase:
+    - "Sim, todos os estados tem mais de 7 letras!"
+    Senão, mostre no console:
+    - "Nem todos os estados tem mais de 7 letras!"
+    */
+    console.log( '\nTodos os estados de `brasil` tem mais de 7 letras?' );
+    var seteLetras = brasil.every(function(item){
+        if ( item.length >= 7){
+            console.log( 'Sim, todos os estados tem mais de 7 letras!' );
```

Comment:
-1
Da forma que você fez, poderia ter usado qualquer método de array que teria o mesmo resultado. A ideia do `every` é verificar o retorno da função que é passada a ele por parâmetro, para saber se todos os elementos retornam true, conforme o que for retornado.

Depois, fora do every, você usa console.log, fazendo a verificação utilizando a variável `seteLetras` :)